scout/adapter/mongo/institute.py

```diff
@@ -65,7 +65,7 @@ def update_institute(
         if not institute_obj:
             raise IntegrityError("Institute {} does not exist in database".format(internal_id))
 
-        updates = {"$set": {}}
+        updates = {"$set": {}, "$pull": {}, "$unset": {}, "$push": {}}
```

Comment:
$unset is missing, this allows you to remove the key completely if the value is empty, I'll run some checks by resetting everything and see what happens